src/data/historikk/historikk_actions.ts

```diff
@@ -1,27 +1,82 @@
-import { Historikk } from "./types/Historikk";
+import { HistorikkEvent, HistorikkKilde } from "./types/historikkTypes";
 
-export function hentHistorikkFeilet(kilde: string) {
-  return {
-    type: `HENT_HISTORIKK_FEILET_${kilde}`,
-  };
+const HENT_HISTORIKK = "HENT_HISTORIKK";
+const HENTER_HISTORIKK = "HENTER_HISTORIKK";
+const HISTORIKK_HENTET = "HISTORIKK_HENTET";
+const HENT_HISTORIKK_FEILET = "HENT_HISTORIKK_FEILET";
+export const HENT_HISTORIKK_FEILET_MOTER = `${HENT_HISTORIKK_FEILET}_MOTER` as const;
+export const HENT_HISTORIKK_FEILET_MOTEBEHOV = `${HENT_HISTORIKK_FEILET}_MOTEBEHOV` as const;
+export const HENT_HISTORIKK_FEILET_OPPFOELGINGSDIALOG = `${HENT_HISTORIKK_FEILET}_OPPFOELGINGSDIALOG` as const;
+export const HENTER_HISTORIKK_MOTER = `${HENTER_HISTORIKK}_MOTER` as const;
+export const HENTER_HISTORIKK_MOTEBEHOV = `${HENTER_HISTORIKK}_MOTEBEHOV` as const;
+export const HENTER_HISTORIKK_OPPFOELGINGSDIALOG = `${HENTER_HISTORIKK}_OPPFOELGINGSDIALOG` as const;
+export const HENT_HISTORIKK_MOTER = `${HENT_HISTORIKK}_MOTER` as const;
+export const HENT_HISTORIKK_MOTEBEHOV = `${HENT_HISTORIKK}_MOTEBEHOV` as const;
+export const HENT_HISTORIKK_OPPFOELGINGSDIALOG = `${HENT_HISTORIKK}_OPPFOELGINGSDIALOG` as const;
+export const HISTORIKK_HENTET_MOTER = `${HISTORIKK_HENTET}_MOTER` as const;
+export const HISTORIKK_HENTET_MOTEBEHOV = `${HISTORIKK_HENTET}_MOTEBEHOV` as const;
+export const HISTORIKK_HENTET_OPPFOELGINGSDIALOG = `${HISTORIKK_HENTET}_OPPFOELGINGSDIALOG` as const;
+
+export interface HentHistorikkFeiletAction {
+  type:
```

Comment:
typeof "enEllerAnnenConst" blir string, så dette blir vel det samme som å skrive type: string?

Kan vurdere å bruke enum

Reply:
Jeg vil at `type` skal være én av tre spesifikke strenger, ikke bare string. Ellers vil ikke reducer forstå hvilken action det er. Jeg fikk i hvert fall problemer med det når jeg brukte enum med streng-verdier 🤔

Reply:
Alternativt kan jeg definere mer spesifikke actions i stedet for at de tar inn kilde.

Reply:
Ser den, men vil det funke? Vil ikke typeof constant bli evaluert til string? Her er eksempel på enum, uten at jeg vet om det er beste løsningen :) https://github.com/navikt/syfomodiaperson/blob/master/src/data/behandlendeenhet/behandlendeEnhet_actions.ts

Reply:
Hmja, fikser en litt enklere løsning med enums.

Reply:
Fikset!